frontend/src/FilterLists/index.tsx

```diff
@@ -37,6 +39,7 @@ export function FilterLists() {
 
   return (
     <>
+      <div style={{ display: 'flex', justifyContent: 'space-between' }}>
```

Comment:
For consistency and maintainability, I suggest moving these styles to a separate CSS file.

Reply:
Thank !